client/src/components/Onboarding/OnboardingAdminList.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { InfiniteScrollTable } from '_common';
+import { v4 as uuidv4 } from 'uuid';
+import './OnboardingAdminList.scss';
+import './OnboardingAdminList.module.scss';
+
+const OnboardingAdminList = ({ users }) => {
+  const columns = [
+    {
+      Header: (
+        <>
+          User
+          <button>Sort</button>
+        </>
+      ),
+      accessor: 'lastName',
+      Cell: el => (
+        <span>
+          {`${el.row.original.firstName} ${el.row.original.lastName}`}
+        </span>
+      )
+    },
+    {
+      Header: 'Step',
+      accessor: 'steps',
+      Cell: el => (
+        <table styleName="cell-table">
+          {el.value.map(step => (
+            <tr>
```

Comment:
This may cause issues. I can make the other columns in the table obey the same table style rendering, but the mouse hover on each individual sub-table row is going to activate independently.

The other solution is to make this column a super-column where the table renders all the other columns.

The last possible solution is to disable all row hovers. The design doesn't really show row hovering anyway, but it is part of InfiniteScrollTable's default behavior.

https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/bd06dcf8-7744-474e-a155-93e3d28fa1a4/

Reply:
@tacc-wbomar

Reply:
Without testing, I believe row hover **can**—or choice—_either_ work on table and sub-table _or_ be disabled for sub-table. If this is the only reason for the investigation into other solutions, I'll help with row hover.

What steps do I need to take to be able to test this locally? (I have a working environment, using a recently freshly created repo and secrets.)

Reply:
We chatted. We should use pagination because (A) it's the design, (B) pagination is available for this table, but also (B) infinite scroll table is _both_ not compatible with pagination _and_ has the above issues that have complex solutions.